src/lib/progress.c

```diff
@@ -153,20 +153,20 @@ static void progress_spinner_print_end(void)
 
 void progress_set_spinner(bool status_flag)
 {
-	if (log_get_level() == LOG_DEBUG || !isatty(fileno(stdout))) {
-		return;
-	}
-
 	if (status_flag) {
 		if (progress_function) {
 			progress_report(-1, 100);
-		} else {
+		} else if (log_get_level() == LOG_DEBUG) {
```

Comment:
It works now, but it's clearer if you use the same if for log_debug and isatty. So you don't need a return here.

Reply:
Done